src/index.js

```diff
@@ -9,5 +9,58 @@ import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
+import Clue from '../src/Clue'
+import '../src/Game'
+import '../src/Player'
+import '../src/Round'
+// import { promises } from 'dns';
 
-console.log('This is the JavaScript entry file - your code begins here.');
+let clue;
+let clueCategories = [];
+let clueInfo = [];
+
+
+
+function categoryFetch() {
+ return fetch('https://fe-apps.herokuapp.com/api/v1/gametime/1903/jeopardy/data')
+  .then(data => data.json())
+  .then(data => data.data.categories)
+  .then(categories => {
+    let catKeys = Object.keys(categories)
+    catKeys.forEach(key => clueCategories.push({[key]: categories[key]}))
+  })
+  .catch(error => console.log('failure'))
+}
+
+
+function clueFetch() { 
+  return fetch('https://fe-apps.herokuapp.com/api/v1/gametime/1903/jeopardy/data')
+  .then(data => data.json())
+  .then(data => data.data.clues)
+  .then(clues => {
+    let clueKeys = Object.keys(clues)
+    clueKeys.forEach(key => clueInfo.push(clues[key]))
+  })
+  .catch(error => console.log('failure'))
+}
+
+function getFetches() {
+  return Promise.all([clueFetch(), categoryFetch()])
+}
+
+
+function instantiateClues() {
+  return clueInfo.map(c => { 
+    clue = new Clue(c)
+    console.log(clue) 
+  })
+}
+
+getFetches()
```

Comment:
Using this and the `Promise.all` is clever! It's keeping your logic here very modular. It would still be a good practice to put a `.catch` at the end of this, in case any of your promises in that array fail.